src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/FlowCreateHubBolt.java

```diff
@@ -172,7 +172,7 @@ public void declareOutputFields(OutputFieldsDeclarer declarer) {
         public FlowCreateConfig(String requestSenderComponent, String workerComponent, int timeoutMs, boolean autoAck,
                                 int flowCreationRetriesLimit, int pathAllocationRetriesLimit,
                                 int pathAllocationRetryDelay, int speakerCommandRetriesLimit) {
-            super(requestSenderComponent, workerComponent, timeoutMs, autoAck);
+            super(requestSenderComponent, workerComponent, null, timeoutMs, autoAck);
```

Comment:
Why you do not add a corresponding argument here? Because of the builder, it can be added without change of the caller side.